docs/docs/setting-up/data-ingestion/from-url.md

```diff
@@ -7,86 +7,87 @@ sidebar_position: 1
+You can find out more about the [helper container in the examples repository](https://github.com/bacalhau-project/examples/tree/main/tools/upload) which is designed to simplify the data uploading process.
+:::
+:::info
+For more details, see the [CLI commands guide](../../dev/cli-reference/all-flags.md#docker-run)
 :::
 
-## Checking the State of your Jobs
+## Checking the State of Your Jobs
 
-- **Job status**: You can check the status of the job using `bacalhau list`.
+- **Job status**: You can check the status of the job using `bacalhau list`, processing the json ouput with the `jq`:
 
 ```bash
-%%bash
 bacalhau list $JOB_ID --output=json | jq '.[0].Status.JobState.Nodes[] | .Shards."0" | select(.RunOutput)'
 ```
 
-When it says `Published` or `Completed`, that means the job is done, and we can get the results.
+When the job status is `Published` or `Completed`, that means the job is done, and we can get the results using the job ID.
 
 - **Job information**: You can find out more information about your job by using `bacalhau describe`.
 
 ```bash
-%%bash
 bacalhau describe  $JOB_ID 
 ```
 
-**Job download**: You can download your job results directly by using `bacalhau get`. Alternatively, you can choose to create a directory to store your results. In the command below, we created a directory and downloaded our job output to be stored in that directory.
+- **Job download**: You can download your job results directly by using `bacalhau get`. Alternatively, you can choose to create a directory to store your results. In the command below, we removed a directory in case it was present before, created it and downloaded our job output to be stored in that directory.
 
 ```bash
-%%bash
 rm -rf results && mkdir ./results
 bacalhau get --output-dir ./results $JOB_ID 
 ```
 
 ## Viewing your Job Output
 
-Each job creates 3 subfolders: the **combined_results**, **per_shard files**, and the **raw** directory. To view the file, run the following command:
+Each job result contains `outputs` subfolder and `extiCode`, `stderr` and `stdout` files with relevant content. To view the execution logs execute following:
 
 ```bash
-%%bash
-head -n 15 ./results/combined_results/stdout
+head -n 15 ./results/stdout
 ```
+And to view the job execution result (`README.md` file in the example case), which was saved as a job output, execute:
+```bash
+tail ./results/outputs/README.md
+```
+
 
 ## Get the CID From the Completed Job
 
 To get the output CID from a completed job, run the following command:
 
 ```bash
-%%bash --out cid
 bacalhau list $JOB_ID --output=json | jq -r '.[0].Status.JobState.Nodes[] | .Shards."0".PublishedResults | select(.CID) | .CID'
 ```
-The job will upload the CID to your public storage via Estuary. We will store the _cid_ that in an environment variable so that we can reuse it later on.
+The job will upload the CID to the public storage via Estuary. We will store the CID in an environment variable so that we can reuse it later on.
```

Comment:
Estuary has recently been deprecated.

```suggestion
The job will upload the CID to the public storage via IPFS. We will store the CID in an environment variable so that we can reuse it later on.
```